docs/install.rst

```diff
@@ -130,33 +130,45 @@ can specify a local install directory to cmake via the
 Library Names
 -------------
 
-By default the installed libraries follow a pattern for how they are
-named. This is done to enable multiple versions of the library to be
-installed and targeted by different builds depending on the needs of
-the project. A simple example of this would be to have different
-versions of the library installed to allow for applications targeting
-different VFX Platform years to co-exist.
+By default, libraries are installed with the following names/symlinks:
 
-If you are building dynamic libraries, once you have configured, built,
-and installed the libraries, you should see the following pattern of
-symlinks and files in the install lib folder:
+.. code-block::
+
+    libOpenEXR.so -> libOpenEXR.so.31
+    libOpenEXR.so.$SOVERSION -> libOpenEXR.so.$SOVERSION.$RELEASE
+    libOpenEXR.so.$SOVERSION.$RELEASE (the shared object file)
+
+The ``SOVERSION`` number identifies the ABI version. Each OpenEXR
+release that changes the ABI in backwards-incompatible ways increases
+this number. By policy, this changes only for major and minor
+releases, never for patch releases. ``RELEASE`` is the
+``MAJOR.MINOR.PATCH`` release name. For example, the resulting shared
+object filename is ``libOpenEXR.so.31.3.2.0`` for OpenEXR release
+v3.2.0. This naming scheme helps reinforce the correspondence between
+the real ``.so`` name and the release it corresponds to, moreso that
+would be indicated through a single increasing number.
+
+By OpenEXR project policy, the additional versioning information in
```

Comment:
Why don't we relegate this paragraph and the link to the libtool manual to the commit message? I don't think there's any need to tombstone libtool here. The release notes could also be the place for the tombstone. I feel like I might get PTSD every time I read the install documentation :)

Reply:
Fine by me, I wanted to leave a record of our logic in making this change, but the PR/commit message is sufficient.